Cargo.lock

```diff
@@ -562,7 +595,7 @@ dependencies = [
  "frame-support",
  "kusama-emulated-chain",
  "parachains-common",
- "sp-core",
+ "sp-core 29.0.0",
```

Comment:
@pepoviola is it possible to deduplicate the dependencies so that we don't have 2 versions of polkadot-sdk crates?

Reply:
or is it coming from subxt?

Reply:
Yes, we can align the version of `sp-core`. There is any reason why we are using `29.0.0` here since the latest release is `32.0.0`?

Thanks!!

Reply:
it's probably fine to leave it as is for now. the versions will be upgraded when we update the runtumes to the next polkadot-sdk release

Reply:
Yea its fine as long as we dont have duplicated versions in the runtimes themselves.